docs/AQL/master03-syntax.adoc

```diff
@@ -479,11 +479,16 @@ WHERE
 AQL provides built-in functions to perform operations on data. Functions are similar to operators as they manipulate data and return a single result, but the difference is in the format and number of their arguments.
 
 In general, a function syntax is `function(argument, argument, ...)`, accepting zero, one or more _arguments_, consisting of <<_constants, constants>>, <<_parameters, parameters>>, <<_variables, variables>>, <<_identified_paths, identified paths>> or other functions.
+Most of these functions are mapped to function definitions in the {openehr_foundation_types}[openEHR Foundation Types^].
```

Comment:
Probably we need to be clear on which ones are defined by those Foundation Types functions, and which ones are not, and why not.

Reply:
I thought also about it, but it is not easy to make a list or an overview that I can integrate here, so I choose to just mention them as a link to that BASE specs.
The second reason was that I tried to avoid coupling too much to other specs - it was always a sensitive aspect of AQL.
Also, these functions are so far is my suggestion - I did not had enough feedback for more functions that should be considered 'core built-in', but I can imagine more will arise during SEC meetings/discussions.

Reply:
I improve the text a bit, perhaps it is now acceptable. As said above, it would be complicated to list which function maps to which other spec-function and why not; I'm not sure about the added value of such mapping table (from the reader perspective) . My impression is that the way the AQL is now specifying and describing functions is good enough to understand and to know what does each function do. Off course, I'm still curious for feedback from SEC in review phase, and we'll change text accordingly.